### PR TITLE
Update to JWT

### PR DESCRIPTION
**Pull request by steffendsommer**
Uses the pre-release of JWT and updates all references from `VaporJWT` to `JWT`. Please validate that the verification process is still correct.

### CODE REVIEW

File: Sources/JWTKeychain/Support/Configuration/Configuration.swift [hunk 1]
```diff
@@ -188,28 +187,19 @@ public struct Configuration: ConfigurationType {
...
-        } catch {
+            // No claims to verify so return true
+            return true
 
+        } catch {
             throw AuthError.invalidBearerAuthorization
-
         }
-
-        return false
```
Comment (pcoutinho) [inline Sources/JWTKeychain/Support/Configuration/Configuration.swift:212]: I think this is missing.

**Review by pcoutinho — COMMENTED** [review]
Not sure if this is missing or I just can't spot it.

**Comment by steffendsommer** [comment]
@Skyback ok, it will now throw (a new type of error) instead of returning a bool. I intentionally added the new error type in order to keep the existing logic on the callsite, although that it might be possible to simplify that as well.

**Comment by pcoutinho** [comment]
@steffendsommer It seems nice :) Just the conflict to resolve and I'm happy to merge it.

**Pull request merged**